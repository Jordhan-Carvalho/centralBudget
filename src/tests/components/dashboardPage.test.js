import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/dashboardPage';

test('should render the dashboard page', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});