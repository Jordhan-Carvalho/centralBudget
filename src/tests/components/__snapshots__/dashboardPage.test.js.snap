// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard page 1`] = `
<Fragment>
  <div
    className="row justify-content-center"
  >
    <div
      className="col-12 boxdash"
    >
      <h3>
        Expenses and Incomes Chart
      </h3>
      <ConnectFunction
        height={200}
      />
    </div>
  </div>
  <div
    className="row justify-content-center"
  >
    <div
      className="col-12 boxdash"
    >
      <h3>
        Savings Chart
      </h3>
      <ConnectFunction
        height={200}
      />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      align="center"
      className="col-12 boxdash"
    >
      <ConnectFunction
        isDashboard={true}
      />
      <hr />
      <ConnectFunction
        type="expense"
      />
      <ConnectFunction
        type="income"
      />
      <ConnectFunction
        type="saving"
      />
    </div>
  </div>
</Fragment>
`;
